### PR TITLE
Added java 8 lambda expressions and also did some code cleanup of catching multiple exceptions in single catch block

### PR DESCRIPTION
**Pull request by gs-athaduka**
Added java 8 lambda expressions and also did some code cleanup of catching multiple exceptions in single catch block

Added java 8 lambda expressions in below classes

AddFindsByCss
AddFindsChildByCss
AddRemoteTouchScreen
AddRotatable
RemoteWebElement

and also did some code cleanup of catching multiple exceptions in single catch block  in below class,

JsonToBeanConverter

- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java [hunk 1]
```diff
@@ -17,25 +17,13 @@
...
 
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonNull;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import com.google.gson.JsonPrimitive;
-import com.google.gson.JsonSyntaxException;
-
+import com.google.gson.*;
```
Comment (jleyba) [inline java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java]: Do not use import *

File: java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java [hunk 2]
```diff
@@ -17,25 +17,13 @@
...
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```
Comment (jleyba) [inline java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java]: Do not use import *

Comment (gs-athaduka) [inline java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java]: This is done. Could you please check now?

File: java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java (hunk 1 above)
Comment (gs-athaduka) [inline java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java]: This is done. Could you please check now?

Comment (gs-athaduka) [inline java/client/src/org/openqa/selenium/remote/JsonToBeanConverter.java]: @jleyba  This is done. Could you please check now?

**Comment by schmidtkp** [comment]
@jleyba I agree with you, in part, "...using lambdas just b/c they're the new hotness...", but I can see some cases where they can reduce redundant code and increase readability. I don't feel lambdas should be used all the time and should only be used with discretion. I can see where lambdas could be abused by obfuscating code, potentially making code less readable. Bottom line is use good judgement.

**Comment by jleyba** [comment]
@schmidtkp I wasn't complaining about lambdas in general. With respect to this PR, I feel the changes for lambdas aren't necessary and don't help readability.

**Comment by gs-athaduka** [comment]
@jleyba  According to me lambda uses invokedynamic to call lambdas thus if you have 10 lambdas it will not result in any anonymous classes thus reducing the final jar size

**Comment by schmidtkp** [comment]
@jleyba I didn't interpret your remarks as complaints. I was just commenting in regards to you wanting to know what others think. We're on the same page.

**Comment by gs-athaduka** [comment]
Any updates on this or should I close this PR for now?

**Comment by shs96c** [comment]
I've been very slow at reviewing PRs. Looking at this now.

**Comment by shs96c** [comment]
Merged as 4f10a2a814a1bfbbae7076ff5dfaaa06db7cb26c .Thank you for the PR, and my apologies for the slow review

**Pull request closed without merging**